getting-started/getting-started.md

```diff
@@ -11,75 +11,71 @@ position: 0
 # Getting Started with Telerik Document Processing
 
 
-Telerik Document Processing includes a set of cross-platform libraries that let you import and export content between different formats and work with archive files. The needed assemblies come with the installation (automated and manual) of several of the Telerik suites and in the internal builds archive where the latter is available. More information about the suites that include Telerik Document Processing and how to install them is available in the [Installing on Your Computer article]({%slug installation-installing-on-your-computer%}). 
+Telerik Document Processing includes a set of cross-platform libraries that let you import and export content between different formats and work with archive files. The needed assemblies come with the installation files of several Telerik suites. They are available also in the internal builds of the same Telerik suites. 
```

Comment:
I don't think that _automatic_ and _manual installation_ are terms that have the same meaning for every reader. So I recommend skipping them.